include/cudaq/Optimizer/Dialect/CC/CCOps.td

```diff
@@ -898,6 +898,68 @@ def cc_AddressOfOp : CCOp<"address_of", [Pure,
   }];
 }
 
+def cc_CreateStateOp : CCOp<"create_state", [Pure] > {
+  let summary = "Create state from data";
+  let description = [{
+    This operation takes a pointer to state data and creates a quantum state.
+    The operation can be optimized away in DeleteStates pass, or replaced
+    by an intrinsic runtime call on simulators.
+
+    ```mlir
+      %0 = cc.create_state %data: !cc.ptr<!cc.state>
```

Comment:
This example only has 1 argument, but below it lists 2 arguments.

Reply:
forgot to update, thanks!

Reply:
done.